### ISSUE TITLE
DID V1.0

### ISSUE THREAD

**Issue by nadalin** [w3c-ccg/did-wg-charter#13]
"Each DID Document contains at least three things: cryptographic material, authentication suites, and service endpoints. Cryptographic material combined with authentication suites provide a set of mechanisms to authenticate as the DID subject (e.g. public keys, pseudonymous biometric protocols, etc.)."

Reword as there should be no authentication discussed in this WG and as I understand it there should be no mechanisms defined here for authentication, signatures, or crypto

**Comment by kimdhamilton** [comment]
we need to reword to say we're enabling those things, not defining, and not necessarily requiring

**Comment by kimdhamilton** [comment]
is #14 a dupe of this?

**Comment by nadalin** [comment]
@kimdhamilton It's different but along the same concerns

**Comment by kimdhamilton** [comment]
I see; those are indeed different statements in the charter. I'll update my spreadsheet.

**Comment by kimdhamilton** [comment]
Note: we need to reword this to say we're enabling those things, not defining, and not necessarily requiring:

> Each DID Document contains at least three things: cryptographic material, authentication suites, and service endpoints. Cryptographic material combined with authentication suites provide a set of mechanisms to authenticate as the DID subject (e.g. public keys, pseudonymous biometric protocols, etc.). Service endpoints enable trusted interactions with the DID subject.

**Comment by msporny** [comment]
The group discussed this on the 2019-06-27 call and agreed on merging PR #45 to address this issue. Closing.
